src/main/java/net/mcreator/ui/component/util/DiscordClient.java

```diff
@@ -40,8 +60,10 @@ public class DiscordClient implements Closeable {
 	private final Timer timer = new Timer();
 
 	public DiscordClient() {
-		if (!PreferencesManager.PREFERENCES.ui.discordRichPresenceEnable)
+
+		if ((!PreferencesManager.PREFERENCES.ui.discordRichPresenceEnable) || (OS.getArchitecture().equals("aarch64"))) {

```

Comment:
You could extract this to a private method isEnabled that returns this check and call method instead